src/doppelkopf.js

```diff
@@ -3,20 +3,14 @@ import getPictureSrc from './utils/getPictureSrc';
 import PlayerSelectHelper from './utils/PlayerSelectHelper';
 import './css/common.css';
 import './css/doppelkopf.css';
+import getScorepads from './utils/getScorepads';
 
 /**
  * The HTML element used to display the list of scorepads.
  */
 let scorepadList;
 
-/**
- * Redirects the user to the specified scorepad.
- *
- * @param {Object} scorepad the scorepad to redirect the user to
- */
-function redirectToScorepad(scorepad) {
```

Comment:
Das hier ist als markiert gelöscht, sollte aber an dieser Stelle bleiben. Gerade ist die Funktion unten lokal in `window.onload` definiert, damit kann kein neues "Doppelkopf"-Spiel mehr erstellt werden:

> ReferenceError: redirectToScorepad is not defined `doppelkopf.js:32:2`